modules/apps/change-tracking/change-tracking-spi/src/main/java/com/liferay/change/tracking/spi/display/BaseCTDisplayRenderer.java

```diff
@@ -93,6 +99,29 @@ public void render(DisplayContext<T> displayContext) throws Exception {
 				displayContext, getResourceBundle(displayContext.getLocale()),
 				themeDisplay));
 
+		User user = (User)httpServletRequest.getAttribute(WebKeys.USER);
+
+		List<Role> roles = user.getRoles();
+
+		boolean userAdmin = false;
+
+		for (Role role : roles) {
+			if (Objects.equals(role.getName(), RoleConstants.ADMINISTRATOR)) {
+				userAdmin = true;
+
+				break;
+			}
+		}
```

Comment:
You can check if the user is an admin with `permissionChecker.isCompanyAdmin`. Permission checker is available in the themeDisplay.